web/css/doc.css

```diff
@@ -30,16 +30,16 @@ body {
     text-decoration: none;
     font-weight: bold;
     padding: 3px;
-    border: dotted #888 1px;
+    border: dotted var(--doc-popuout-a-border) 1px;
     border-radius: 5px;
 }
 
-.md pre.tilde.clickable {
-    background-color: #f0f8ff;
+.md pre.tilde {
+    background-color: var(--doc-tilde-bg) !important;
```

Comment:
Use of `!important` is sometimes bad for accessibility, since it overrides customizations in user style sheets.  Can you get this to take priority by being more specific, instead?

Same for other uses below.

Reply:
In places where `!important` used I override inline styles applied by `markdeep`. This happens after loading of doc.css, and have higher priority.
What means more specific?

Reply:
Okay, that makes sense.  What I meant was that it's possible to make one CSS selector override another by making it more specific.  For example, a `div.foo` rule will always take precedence over a  `foo` rule, if they give different values for the same property.

Unfortunately, though, there is no way to take precedence over an inline style on an HTML element.  It's always considered the most specific rule, and take priority over any selector.  So use of `!important` here is the right choice.  Thanks for explaining.